server/remote/msgraph/subscription.go

```diff
@@ -0,0 +1,82 @@
+// Copyright (c) 2019-present Mattermost, Inc. All Rights Reserved.
+// See License for license information.
+
+package msgraph
+
+import (
+	"fmt"
+	"net/http"
+	"time"
+
+	"github.com/mattermost/mattermost-plugin-msoffice/server/remote"
+)
+
+const subscribeTTL = 10 * time.Minute
+
+func (c *client) createSubscription(resource, changeType, notificationURL string) (*remote.Subscription, error) {
+	sub := &remote.Subscription{
+		Resource:           resource,
+		ChangeType:         changeType,
+		NotificationURL:    notificationURL,
+		ExpirationDateTime: time.Now().Add(subscribeTTL).Format(time.RFC3339),
+	}
+	err := c.rbuilder.Subscriptions().Request().JSONRequest(c.ctx, http.MethodPost, "", sub, sub)
+	if err != nil {
+		return nil, err
+	}
+	c.LogDebug("msgraph: created subscription", "subscriptionID", sub.ID, "resource", sub.Resource, "changeType", sub.ChangeType)
+	return sub, nil
+}
+
+func (c *client) CreateEventMessageSubscription(notificationURL string) (*remote.Subscription, error) {
+	return c.createSubscription(
+		// TODO make work: "me/messages?filter=microsoft.graph.eventMessage/meetingMessageType ne ''",
```

Comment:
Should there be a ticket for this TODO?

Reply:
I removed the support for EventMessages for the time being. They look promising in that there's more information there, but it's a lot simpler to stick to Events for now.